include/mbedtls/config.h

```diff
@@ -2254,6 +2254,19 @@
  */
 #define MBEDTLS_ECP_C
 
+/**
+ * \def MBEDTLS_USE_UECC
+ *
+ * Enable the tinycrypt ECC library.
+ *
+ * Module:  tinycrypt/ecc.c
+ *          tinycrypt/ecc_dh.c
+ *          tinycrypt/ecc_dsa.c
+ *
+ * This module provides alternative ECC handling functions
```

Comment:
Very (very) minor: Dot missing at the end of the line.

Reply:
Done